src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+	struct RangePayment {
+        IStrategy[] strategies;
+        // relative weights of shares in the strategies
+        uint256[] weights;
+        IERC20 token;
+        uint256 amount;
+        uint256 startTimestamp;
+        // duration in seconds
+        uint256 duration;
+	}
+
+    struct DistributionRoot {
+        // timestamp after which the root can be claimed against
+        uint32 activatedAfter;
+        // merkle root of the distribution
+        bytes32 root;
+    }
+
+    struct MerkleLeaf {
+        IERC20 token; 
+        uint256 amount;
+        // Explicit recipient of the claim
+        address recipient;
+    }
+
+    struct PaymentMerkleClaim {
```

Comment:
This should likely be 

```solidity
struct PaymentMerkleClaim {
        MerkleLeaf[] leaves;
        // The index of the root in the list of roots
        uint32 rootIndex;
        bytes accountTreeProof;
        // The index of the leaf in the merkle tree for this root
        uint32[] leafIndices;
        bytes[] tokenTreeProofs;
    }
```

or something to enable ammortizing the account tree proof when claiming multiple tokens

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/470/commits/1d438f42e18ff6e494ed572df71b024a4e965de6
I'm fine with this for now

Reply:
this should also have an `accountTreeIndex` field, I think.

Reply:
added in https://github.com/Layr-Labs/eigenlayer-contracts/pull/470/commits/f4a3836b095ff5e45823afe0cec3b42106f9c109
resolving this thread; a new one can be opened for further changes